src/wasm-linker.cpp

```diff
@@ -366,10 +369,30 @@ bool hasI64ResultOrParam(FunctionType* ft) {
   return false;
 }
 
+void Linker::makeDummyFunction() {
+  bool create = false;
+  // Check if there are indirectly called functions
```

Comment:
is it really impossible to get a comparison to a function pointer without this? couldn't one compare `(void*)17` to `nullptr` for example? I guess that's a weird thing we don't care about much ;) but i wonder if we shouldn't be totally general.